cmd/skaffold/app/cmd/fix.go

```diff
@@ -121,8 +121,16 @@ func fix(out io.Writer, configFile, outFile string, toVersion string) error {
 	if outFile != "" {
 		var mvErr error
 		if overwrite {
+			if err != nil {
```

Comment:
It looks like we don't need to check this error here? We are not updating its value. If that's the case let's remove the condition.